rules/kubelet-event-qps/raw.rego

```diff
@@ -0,0 +1,34 @@
+package armo_builtins
+
+import future.keywords.in
+
+# if --event-qps is present rule should pass
+
+# --event-qps argument is not present
+deny[msga] {
+ 	
+    kubelet_info := input[_]
+    kubelet_info.kind == "KubeletInfo"
+    kubelet_info.apiVersion == "hostdata.kubescape.cloud/v1beta0"
+    command := kubelet_info.data.cmdLine
+    
+    not contains(command, "--event-qps")
```

Comment:
What about `--config`?

Reply:
I have added that check but github does not indicate the change